source/_docs/domains.md

```diff
@@ -0,0 +1,182 @@
+  <thead class="thead-inverse">
+    <tr>
+      <th scope="row" class="thead-inverse"></th>
+      <th>Personal</th>
+      <th>Professional</th>
+      <th>Business</th>
+      <th>Elite</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th scope="row" class="thead-inverse">Custom Domain Limit (per site)</th>
+      <td>5</td>
+      <td>25</td>
+      <td>100</td>
+      <td>200</td>
+    </tr>
+    <tr>
+      <th scope="row" class="thead-inverse">Free and managed HTTPS</th>
+      <td>✓</td>
+      <td>✓</td>
+      <td>✓</td>
+      <td>✓</td>
+    </tr>
+  </tbody>
+</table>
+
+<div class="alert alert-info" role="alert">
+<h4 class="info">Note</h4>
+<p markdown="1">Add all domains you want to resolve to Pantheon within the Site Dashboard for each respective environment, as described in [Launch Essentials](/docs/guides/launch/). Automatic resolution of domains and wildcards are not supported.</p></div>
+
+### Primary domain
+A primary domain refers to a single custom domain used to serve responses, regardless of originating URL of a request (e.g., `https://www.example.com`). Configure the primary domain as part of the going live process described in [Launch Essentials: Redirect to a Primary Domain](/docs/guides/launch/redirects/).
+
+We recommend using the www subdomain prefix for all sites. While it’s good for visitors and DNS to resolve both www and the domain itself, it's best practice to choose one or the other and redirect from www to non-www (or vice versa, your call). This optimizes SEO by avoiding duplicate content and prevents session strangeness, where a user can be logged in to one domain but logged out of other domains at the same time.
+
+## Vanity domains for organizations
+Pantheon Partners, Strategic Partners, Enterprise accounts, Resellers, and OEM Partners have the ability to replace `pantheonsite.io` for each environment on every site they run with a custom vanity domain.
+
+For details, see [Vanity Domains](/docs/vanity-domains/).
+
+## Redirects
+It's often useful to redirect requests to a different domain or path. While it's technically possible to use Drupal or WordPress to perform the redirect, it's faster and more efficient to redirect without having to fully bootstrap your web application.
+
+<div class="alert alert-info">
+<h4 class="info">Note</h4>
+<p markdown="1">Redirects should be managed in PHP, since `.htaccess` is ignored. For details, see [Using PHP as an htaccess Alternative](/docs/htaccess/).</p>
+</div>
+
+### Command line conditionals
+All redirect logic run on Pantheon environments should include the `php_sapi_name() != "cli"` conditional statement to see if WordPress or Drupal is running via the command line. Otherwise, redirects kill the PHP process before Drush and WP-CLI is executed resulting in a silent failure:
+
+```bash
+[notice] Command: <site>.<env> -- 'drush <command>' [Exit: 1]
+[error]
+```
+
+### Redirect to HTTPS and the primary domain
```

Comment:
We're managing the contents of the snippets in a single source, and inserting it on both pages at build time. I think it's fine on both pages but defer to you. Let me know if you still think we should remove and link to the guide @ari-gold